examples/neural_networks/plot_rbm_logistic_classification.py

```diff
@@ -41,8 +41,9 @@
 from sklearn.base import clone
```

Comment:
Perhaps we could move the imports to the cells in which they are first used.

Reply:
Very good point, thank you!